spec/actions/app_spec.cr

```diff
@@ -28,37 +28,37 @@ module DppmRestApi::Actions::App
     end
     describe (fmt_route "/:app_name/service/boot") do
       pending "responds with 401 Forbidden" do
-        put (pp! fmt_route "/some_app/service/boot")
+        put (fmt_route "/some_app/service/boot")
```

Comment:
Please, focus on the PR title to keep changes as minimal as possible - integrate the CLI.